### ISSUE TITLE
How to run the bot multiple times at the same time on the same laptop?

### ISSUE THREAD

**Issue by git-creat** [paulmaunders/delivery-slot-bot#76]
Good afternoon, does anyone have any idea how to run the bot multiple times at the same time on the same laptop?

I'm using Visual Studio Code, I tried opening a new terminals to run in multiple times but it doesn't work, it only allows me to run it ones.

I want to do this because there are 7 Tesco click and collect locations and 1 Asda home deliver location where I live, the bot can only check one click and collect location at a time I think, so I need to make new Tesco accounts and run another copy of the bot to check the other click and collect locations.

Where I live Asda has home delivery and no click and collect, but tesco is the opposite, they have click and collect but no home delivery, in the config.ini file I can only have it on home delivery or click and collect or the ASDA/Tesco websites cause the program to crash with error messages, so I need to be able to run two copies of the bot to check them separately.

**Comment by obbardc** [comment]
yeah, tesco seems to only show on the page 5 collection locations on the click-collect page.
but i think you can change the `locationId` in the URL to any store ?
